lib/ansible/module_utils/linode.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
module_utils files should be licensed as BSD

Reply:
OK, will fix this up then. Thanks.